GitUp/Application/Base.lproj/MainMenu.xib

```diff
@@ -1140,7 +1140,7 @@ UI design by Wayne Fan</string>
                     <box boxType="custom" borderType="none" borderWidth="0.0" cornerRadius="10" title="Box" titlePosition="noTitle" id="UoN-n1-xSr">
                         <rect key="frame" x="0.0" y="0.0" width="280" height="349"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
-                        <view key="contentView" id="UVs-2r-EAT">
+                        <view key="contentView" id="UVs-2r-EAT" customClass="GIWelcomeWindowDestinationView">
```

Comment:
Sorry, very last detail that I missed. `GI` is generally reserved for things that are part of GitUpKit. Because this is in the GitUp project level, we should probably change it to simply be `WelcomeWindowDestinationView`. I believe that's it and then we should be ok to merge this baby in 😄.

Reply:
Fixed. :)